src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -88,6 +88,14 @@ enum xgq_cmd_clock_req_type {
 	XGQ_CMD_CLOCK_SCALE		= 0x2,
 };
 
+/**
+ * clock scaling request types
+ */
+enum xgq_cmd_clock_scaling_req_type {
+	XGQ_CMD_CLK_SCALING_GET_STATUS	= 0x1,
+	XGQ_CMD_CLK_SCALING_SET_OVERRIDE= 0x2,
+};
```

Comment:
Hi @xdavidz ,
if possible please remove "enum xgq_cmd_clk_scaling_app_id" in this file and replace all _AID_* enums usage in xgq.c also with this new definitions. Otherwise, I will take care of it once this PR is merged. Thank you.